### PR TITLE
Add Auto-FontSize to Recommendation component

### PR DESCRIPTION
**Pull request by timtong1982**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x ] Include a change request file using `$ npm run change`

#### Description of changes

Add Auto-FontSize to Dashboard/Recommendation for auto-sizing the font in the header in order to keep the header text in one line for stylish

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by samiyaakhtar** [comment]
I would suggest making this change in the card frame itself. Why do we need auto-sizing? @venkatna I thought it was decided on Friday that we will truncate the overflow text

**Comment by samiyaakhtar** [comment]
Ideally would be nice to use the Header component under Card (or update that one to use this way), unless the design requires them to have different red lines. cc @venkatna

**Comment by venkatna** [comment]
I agree. How do these two differ in red lines?

---
In reply to: [407499573](https://github.com/OfficeDev/office-ui-fabric-react/pull/5666#issuecomment-407499573) [](ancestors = 407499573)

File: packages/dashboard/src/components/Recommendation/__snapshots__/Recommendation.test.tsx.snap [hunk 1]
```diff
@@ -1512,8 +1528,16 @@ exports[`RecommendationCard renders imageillustration example correctly 1`] = `
...
-             
+            <p
+              style={
+                Object {
+                  "fontSize": 28,
+                  "width": 0,
+                }
+              }
+            >
+              Set user passwords to never expire
```
Comment (siddarthkanted) [inline packages/dashboard/src/components/Recommendation/__snapshots__/Recommendation.test.tsx.snap]: can we use localised strings?

**Comment by dzearing** [comment]
Per convo offline, closing and @timtong1982 will resubmit after pulling latest master.

**Pull request closed without merging**